extensions/Worker.Extensions.Storage.Blobs/src/BlobStorageConverter.cs

```diff
@@ -105,6 +107,17 @@ private async ValueTask<ConversionResult> ConvertFromCollectionBindingDataAsync(
 
                 return ConversionResult.Success(result);
             }
+            catch (JsonException ex)
```

Comment:
Are we only making this change to Blob extension converter? We have a built-in JSonPocoConverter as well. Wonder we should do similar thing there as well.

Reply:
That probably is a breaking change. So may be something we can discuss with the team and address later in a separate PR.

Reply:
Yeah this PR specifically focuses on the use case for blob for POCO serialization. We can address this in another PR